css/theme/main.css

```diff
@@ -3848,3 +3848,4 @@ body {
 
 .title {
   text-shadow: 0px 0px 15px black;
+}
```

Comment:
There was also just a missing bracket? Idk why this wasn't here lol

Reply:
LOL nice catch